### PR TITLE
small genomes too

### PR DESCRIPTION
**Pull request by afshinfard**
As in our latest experiments abyss is doing just as nice on smaller genomes, we decided to include in the description that it does work on smaller genomes too.

### CODE REVIEW

**Comment by afshinfard** [comment]
Also, the `biostar` links does not work anymore. We can fix them too. Do I change to the Biostar homepage or that was some sort of a feature still available but in a different way?

**Comment by lcoombe** [comment]
> Also, the `biostar` links does not work anymore. We can fix them too. Do I change to the Biostar homepage or that was some sort of a feature still available but in a different way?

I think it just needs to be updated to https://www.biostars.org/tag/abyss/ ?

**Comment by afshinfard** [comment]
> I think it just needs to be updated to https://www.biostars.org/tag/abyss/ ?

Right, and I also removed `/p/` in the other link and it is working now.

**Comment by afshinfard** [comment]
@lcoombe I do not have access to add reviewers, labels, etc, and also to merge. If you think this PR is ready let's make it to the master. Please note I edited the intro sentence one more time to make sure the message is clear that abyss is really good also for smaller genomes; sounds good?

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ABySS
 =====
 
 ABySS is a *de novo* sequence assembler intended for short paired-end
-reads and large genomes.
+reads. Initially designed for large genomes, ABySS also performs a decent job assembling small (e.g. bacterial) genomes.
```
Comment (lcoombe) [inline README.md:10]:
```suggestion
reads. Initially designed for large genomes, ABySS also performs well on small (e.g. bacterial) genomes.
```
Or some thing like that? I wasn't sure about the phrase "decent job"

Comment (afshinfard) [reply README.md:10]: My only concern is that people may read this and think "well, they mean it does work on smaller genomes but probably they're not the best". But we know we're doing as good as other alternatives... so how do you think we can make this impression?

Comment (afshinfard) [reply README.md:10]: @lcoombe, I've seen that on some well-known repos people add a comparison too.. like a small table showing their results side-by-side of another tool well-used by the community... that'd show everything but I am not sure if we like that sort of thing in a repo readme

Comment (sjackman) [reply README.md:10]: ABySS is a *de novo* sequence assembler intended for short paired-end reads and genomes of all sizes.

Comment (lcoombe) [reply README.md:10]: Thanks Shaun - I like this better!
I think updating that text is fine for now, if desired we could add more in a future PR.

Comment (afshinfard) [reply README.md:10]: Thanks @sjackman, I like your idea! It's simple and right! we don't need to give a background of what abyss was designed for in here!

**Pull request merged**